### DISCUSSION TITLE
[Contract Verification] <Thirdweb>

### DISCUSSION THREAD

**Discussion by zkbudz** [zkSync-Community-Hub/zksync-developers#64] [Contract Verification] [answered]
### Environment

Mainnet

### Issue Type

- [ ] Plugin
- [ ] Block Explorer

### Contract Address

0x7547eDaFdc292E25AE9556451cD77F81dB0E5A75

### Compiler Type

Single file

### zkSolc Version

v1.3.13

### Solc Version

0.8.20

### Contract Name

ZKbudz Harvesters DAO Token

### Contract Code

third web 

### Constructor Arguments

third web

**Comment by MexicanAce** [comment, marked as answer]
Hi @zkbudz 

You should be able to verify your smart contract via the Block Explorer website: https://explorer.zksync.io/address/0x7547eDaFdc292E25AE9556451cD77F81dB0E5A75#contract

Please let us know if you have any questions or concerns

**Comment by amyliasnow** [comment]
To verify a contract on Etherscan, you need to provide the following:

Contract Address: 0x7547eDaFdc292E25AE9556451cD77F81dB0E5A75
Compiler Version: Solidity 0.8.20
Optimization: Enabled or Disabled (if enabled, provide the optimization runs)
Constructor Arguments: Please provide the constructor arguments used when deploying the contract
Contract Name: ZKbudz Harvesters DAO Token
With this information, you can use the appropriate verification command or plugin to verify your contract on Etherscan. Make sure to check the correctness of the provided details before proceeding with the verification process.

**Comment by pobertester** [comment]
Environment: Mainnet
Issue Type: Plugin (Hardhat Verify)
Block Explorer: Contract Address 0x7547eDaFdc292E25AE9556451cD77F81dB0E5A75
Compiler Type: Single file
zkSolc Version: v1.3.13
Solc Version: 0.8.20
Contract Name: ZKbudz Harvesters DAO Token
Constructor Arguments: third web

**Comment by reginaldolivas** [comment]
Contract ZKBudzHarvestersDAO Token
 Compiler version: v0.8.20

 SPDX-License-Identifier: MIT
pragma solidity ^0.8.20;

contract ZKBudzHarvestersDAO 
     Contract implementation

**Comment by postm4ltolin** [comment]
Unfortunately, it's not clear what the specific issue is from the given information. It appears to be related to verifying a contract on the mainnet using zkSolc and possibly encountering an error with the plugin or block explorer. Without more details or error messages, it's difficult to provide a solution.

**Comment by cioprogephen** [comment]
The user is encountering an issue with a plugin or block explorer related to a contract with the address 0x7547eDaFdc292E25AE9556451cD77F81dB0E5A75 on the Mainnet. The contract name is "ZKbudz Harvesters DAO Token," and the code is provided via a third-party web source. The user has not provided any details on the specific issue or error they are encountering, and no repo link or information on the Hardhat Verify Plugin version is provided. The contract was compiled using zkSolc version v1.3.13 and Solc version 0.8.20.
